### PR TITLE
Test failure event

### LINKED ISSUE #26: When a test fails, the test should have a TEST_FAILURE event.

### PR DESCRIPTION
**Pull request by tonyredondo**
Closes #26 

Adds test failure event.

### CODE REVIEW

File: testing.go [hunk 1]
```diff
@@ -93,6 +93,10 @@ func (test *Test) End() {
 	if test.t.Failed() {
 		test.span.SetTag("test.status", "FAIL")
 		test.span.SetTag("error", true)
+		test.span.LogFields(
+			log.String(EventType, EventTestFailure),
+			log.String(EventMessage, "Test has failed"),
```
Comment (fermayo) [inline testing.go]: We don’t have the failure message?

Comment (tonyredondo) [reply testing.go]: I guess not.

I have only the boolean:
image

Maybe if we in the future intercept the `Fatal` and `Fatalf` we can use that message. But the `*testing.T` doesn't have that info

Comment (tonyredondo) [reply testing.go]: Because the `Fail()` method doesn't have any argument for a reason.

Comment (fermayo) [reply testing.go]: Ok, good enough for now

**Pull request merged**